### ISSUE TITLE
Problems connecting to the pool in the latest version lolMiner 1.44

### ISSUE THREAD

**Issue by claudiodelf** [Lolliedieb/lolMiner-releases#1180]
Problems connecting to the pool in the latest version lolMiner 1.44
Last night as soon as the new version was released on hive.os I updated and started the rig. This morning I saw that the rig was working but when I go to see the earnings in the toncoin pool https://next.ton-pool.com I see that it had been disconnected from the pool for more than 9 hours ..... In my opinion the previous version was better! This one has problems for me.

**Comment by hender1964** [comment]
I am also seeing something similar.  I am using ton-pool.com as well.  v1.43 worked great.  I upgraded to 1.44 when it was released in HiveOS.  Two hours later I checked pool hashrate and it was zero.  The miner showed 12.1GH/s and 10.9GH/s (two 3070 non-LHR rigs) but both indicated 0 pool hashrate pool.  I switched back to 1.43 and it worked as expected thru the night.  I decided to try it again today.  After an hour the pool hash rate had dropped significantly so I switched back to 1.43.  Seems like 1.44 is not working with ton-pool.com.
